application/templates/static_site/_newsletter-sign-up.html

```diff
@@ -1,23 +1,15 @@
-<div class="newsletter-sign-up">
-  <div class="grid-row">
```

Comment:
@TheDoubleK it does still have a grid system (see https://design-system.service.gov.uk/styles/layout/#grid-system ) but here we don't need to use that because it's full width so it gets the width of its container anyway.